<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metrics_py</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(115, 0, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>60</y>
      <width>881</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="history">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>881</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Day</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Month</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Serial</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hour</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>History</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>buscar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>236</x>
        <y>3</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="fai_history">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>31</y>
        <width>891</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fail_name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Serial</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Fail history</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>214</x>
        <y>0</y>
        <width>62</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>buscar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>3</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="yield_global">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>33</y>
        <width>881</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Pass</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fails</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Yield</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Yield global</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>214</x>
        <y>1</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>buscar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>3</y>
        <width>221</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableWidget" name="yield_day">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>34</y>
        <width>881</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pass_qty</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fail_qty</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Yield</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Yield day</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>2</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>buscar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>5</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>801</width>
        <height>501</height>
       </rect>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-10</y>
      <width>891</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0.953271, y2:0.216, stop:0.284404 rgba(0, 76, 251, 255), stop:0.688073 rgba(0, 216, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="bt_history">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>96</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>History</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_failhistory">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>20</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Fail history</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_yiel_global">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Yield global</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_yiel_day">
     <property name="geometry">
      <rect>
       <x>322</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Yield day</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_metrics">
     <property name="geometry">
      <rect>
       <x>422</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(146, 146, 146, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Metrics</string>
     </property>
    </widget>
    <widget class="QLabel" name="ffix">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
